### PR TITLE
BZ#2010362 - Update order of Grafana Dashboard description

### PR DESCRIPTION
**Pull request by emarcusRH**
Fixes issue [BZ#2010362](https://bugzilla.redhat.com/show_bug.cgi?id=2010362)
 RHV 4.4.9-1 or later
Changes proposed in this pull request:
- Update the sequential order of elements in each Grafana dashboard
in Administration Guide > Monitoring and observability > Grafana Dashboards

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @emarcusRH )

This pull request needs review by:

### CODE REVIEW

**Review by avlitman — APPROVED** [review]

**Review by avlitman — APPROVED** [review]
/lgtm

File: source/documentation/administration_guide/topics/Grafana_dashboards.adoc [hunk 1]
```diff
@@ -10,37 +10,39 @@ The following dashboards are available in the initial Grafana setup to report Da
 | Dashboard type |  Content
 | *Executive dashboards*
     a|
-    * Cluster dashboard - resource usage, peaks, over-commit, and up-time for hosts and virtual machines in a selected cluster, according to the latest configurations.
+    * System dashboard - resource usage and up-time for hosts and storage domains in the system, according to in the latest configurations.
```
Comment (stoobie) [inline source/documentation/administration_guide/topics/Grafana_dashboards.adoc:13]:
```suggestion
    * System dashboard - resource usage and up-time for hosts and storage domains in the system, according to the latest configurations.
```

**Review by stoobie — CHANGES_REQUESTED** [review]
One tiny change requested.

File: source/documentation/administration_guide/topics/Grafana_dashboards.adoc (hunk 1 above)
Comment (avlitman) [reply source/documentation/administration_guide/topics/Grafana_dashboards.adoc:13]: +1

**Review by sandrobonazzola — APPROVED** [review]

**Review by stoobie — APPROVED** [review]
LGTM

**Pull request merged**